code/game/jobs/job/exploration_vr.dm

```diff
@@ -130,6 +130,7 @@ var/const/SAR 				=(1<<14)
 	outfit_type = /decl/hierarchy/outfit/job/assistant/explorer
 	job_description = "Off-duty crew has no responsibilities or authority and is just there to spend their well-deserved time off."
 	pto_type = PTO_EXPLORATION
+	economic_modifier = 6
```

Comment:
This should be five, on account of pilot

Reply:
Ah, I missed the pilot. Lowering!